src/set.rs

```diff
@@ -229,11 +229,11 @@ fn example() -> Result<(), Box<dyn std::error::Error>> {
     while let Some((v, s)) = stream.next() {
         vs.push((String::from_utf8(v.to_vec())?, s));
     }
-    // Currently, there isn't much interesting that you can do with the states.
+    
     assert_eq!(vs, vec![
-        ("foo".to_string(), Some(183)),
-        ("foob".to_string(), Some(123)),
-        ("fozb".to_string(), Some(83)),
+        ("foo".to_string(), Some(LevenshteinState { state_idx: 183, distance: Some(0) })),
+        ("foob".to_string(), Some(LevenshteinState { state_idx: 123, distance: Some(1) })),
+        ("fozb".to_string(), Some(LevenshteinState { state_idx: 83, distance: Some(2) })),
```

Comment:
Please wrap lines to 79 columns inclusive.

Reply:
I'll make all changed lines shorter.

FYI: When you add `error_on_line_overflow = true` and `error_on_unformatted = true` into `rustfmt.toml` and run `cargo +nightly fmt --all`, it reports 43 errors.